css/carousel.css

```diff
@@ -0,0 +1,141 @@
+.carousel {
+    margin-bottom: 100px
+  }
+  
+  .carousel-container {
+    width: 96%;
+    height: 400px;
+    margin: 0 auto;
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    border-radius: 10px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    transition: box-shadow 0.3s;
+  }
+  
+  .carousel-container:hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    animation: size-up 200ms forwards;
+  }
+  
+  @keyframes size-up {
+    100% {
+      transform: scale(1.005);
+    }
+  }
+  
+  #previous {
+    position: absolute;
+    top: 40%;
+    left: 0px;
+    z-index: 99;
+    background-color: transparent;
+    border: none;
+    border-radius: 50%;
+    color: white;
+    padding: 12px 16px;
+    font-size: 20px;
+    cursor: pointer;
+  }
+  
+  #next {
+    position: absolute;
+    top: 40%;
+    right: 0px;
+    z-index: 99;
+    background-color: transparent;
+    border: none;
+    border-radius: 50%;
+    color: white;
+    padding: 12px 16px;
+    font-size: 20px;
+    cursor: pointer;
+  }
+  
+  #next:hover, #previous:hover{
```

Comment:
If you were to go with a common class for `carousel-buttons` like in the previous comment, then this rule could also target that class instead of both buttons explicitly. 😉